### DISCUSSION TITLE
Warning - Use of this tool exposes you to potential security threats

### DISCUSSION THREAD

**Discussion by AdnanJusicMS** [SharePoint/sp-dev-docs#8417] [Q&A] [answered]
Hi All,

I have a customer which asked me regarding the following message (message pops up in F12 Developer Tool during the page load), which occurs by loading a modern SharePoint Site Collection Page, is this something serious or not and should we concern about security on this page based on this message?
 
![dd](https://user-images.githubusercontent.com/96141055/188873956-87b88f5f-5897-4951-b613-26bd3b181e04.png)

I have done some research and found out, that this happens only on Site Collections with Modern UI, on of this site collections have also custom code deployed by SPFX Apps.

Based on some researches, I found the following information, which claims to ignore the message.

https://github.com/OlivierCC/spfx-40-fantastics/issues/45
https://github.com/pnp/sp-dev-fx-webparts/issues/235

Does anyone faced the same issue?

Thanks a lot in advance!

Kind regards,

Adnan

**Comment by AJIXuMuK** [comment, marked as answer]
Hi @AdnanJusicMS - this is the standard warning provided by SharePoint Framework.
It informs users that they have third part web parts running on their pages.
And these 3p web parts have access to SharePoint data.

I would think of this warning as the popup in Windows when you're trying to run some 3p app for the first time.
